packages/runner-ct/src/iframe/iframes.jsx

```diff
@@ -18,28 +18,36 @@ export function getSpecUrl ({ namespace, spec }, prefix = '') {
 
 @observer
 export default class Iframes extends Component {
-  _disposers = []
-  containerRef = null
+  _disposers = [];
+  containerRef = null;
 
   constructor (props) {
     super(props)
   }
 
   render () {
-    const { height, width, scriptError } = this.props.state
+    const {
+      height,
+      width,
+      scale,
+      scriptError,
+    } = this.props.state
 
     return (
-      <div className={cs('iframes-ct-container', { 'has-error': !!scriptError })}>
+      <div
+        className={cs('iframes-ct-container', { 'has-error': !!scriptError })}
+      >
         <div
-          ref={(container) => this.containerRef = container}
-          className='size-container'
+          ref={(container) => (this.containerRef = container)}
+          className="size-container"
           style={{
             height,
             width,
+            transform: `scale(${scale})`,
           }}
         />
         <ScriptError error={scriptError} />
-        <div className='cover' />
+        <div className="cover" />
```

Comment:
it's a little thing but I think we should avoid changes like this that are not strictly part of the PR - if we want to have a specific style we need linter rules, it makes the PR really noisy